consensus/src/rand/rand_gen/rand_manager.rs

```diff
@@ -170,6 +171,12 @@ impl<S: TShare, D: TAugmentedData> RandManager<S, D> {
             .iter()
             .flat_map(|b| b.ordered_blocks.iter().map(|b3| b3.round()))
             .collect();
+        fail_point!("rand_manager::process_ready_blocks", |_| {
```

Comment:
is this change related?

Reply:
fixed

Reply:
actually that's needed by the new smoke test. Bringing it back.